### PR TITLE
Improve try-state docs

### PR DESCRIPTION
**Pull request by Szegoo**
Part of: #13796

### CODE REVIEW

**Comment by Szegoo** [comment]
@liamaharon PTAL.

File: utils/frame/try-runtime/cli/src/lib.rs [hunk 1]
```diff
@@ -336,14 +336,14 @@
 //!     -lruntime=debug \
 //!     execute-block live \
 //!     --try-state System,Staking \
-//!		--pallet System,Staking \
+//! 		--pallet System,Staking \
```
Comment (liamaharon) [inline utils/frame/try-runtime/cli/src/lib.rs:339]:
```suggestion
//! 	--pallet System,Staking \
```

**Review by liamaharon — APPROVED** [review]
Thanks.

Did you check there were no similar issues with the other documented commands?

**Comment by Szegoo** [comment]
> Did you check there were no similar issues with the other documented commands?

Yeah made some fixes. I have run all the commands locally, it should be good now.

**Review by bkchr — APPROVED** [review]

**Pull request merged**